staging/src/k8s.io/apimachinery/go.mod

```diff
@@ -13,7 +13,7 @@ require (
 	github.com/golang/groupcache v0.0.0-20160516000752-02826c3e7903
 	github.com/golang/protobuf v1.3.2
 	github.com/google/go-cmp v0.3.0
-	github.com/google/gofuzz v1.0.0
+	github.com/google/gofuzz v1.0.1-0.20191028174853-db92cf7ae75e
```

Comment:
unless there's a good reason not to, we prefer to use semver-tagged releases (it helps ensure we are consuming a version the library author intended to be used)

you will also need to bump this by following the instructions at https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/vendor.md#adding-or-updating-a-dependency, which are basically:

```
hack/pin-dependency.sh github.com/google/gofuzz $new-version
hack/update-vendor.sh
<commit>
```

Reply:
done

Reply:
> unless there's a good reason not to, we prefer to use semver-tagged releases (it helps ensure we are consuming a version the library author intended to be used)

This is still outstanding. It doesn't look like there's a tagged version of this library yet that has the changes you want. In this scenario, we usually reach out to the upstream component to ask what the timeframe for a tagged release is.

Reply:
This snippet is out of date